pkg/graph/schema.resolvers.go

```diff
@@ -1511,6 +1511,36 @@ func (r *queryResolver) UserAccount(ctx context.Context, username string) (*auth
 	return resolvers.UserAccountGetByUsername(r.store, username)
 }
 
+// SystemWorkspace is the resolver for the systemWorkspace field.
+func (r *queryResolver) SystemWorkspace(ctx context.Context, cedarSystemID string) (*model.SystemWorkspace, error) {
+	g := new(errgroup.Group)
```

Comment:
thoughts on this? i opted for this because SystemWorkspace will eventually be getting a ton of data from a ton of places all at once. it seemed best to start in a scalable way

Reply:
Interesting approach! Is there a difference between this (a single resolver that calls both `store` methods in go routines) and writing a resolver for each request type?

Both would allow concurrently fetching related requests, but the latter would mean you don't _always_ have to fetch both (like in the case you only ask for one piece of information)

Reply:
this has been refactored!